ci-operator/jobs/openshift/dpu-operator/openshift-dpu-operator-main-presubmits.yaml

```diff
@@ -5,22 +5,25 @@ presubmits:
     branches:
     - ^main$
     - ^main-
-    cluster: build04
```

Comment:
Why are you removing the image builder test?

Reply:
this was autogenerated.

Reply:
Do not update the job config manually